lib/animation/Animation.js

```diff
@@ -76,7 +76,18 @@ function Animation(canvas, eventBus) {
   });
 
   eventBus.on(PROCESS_INSTANCE_FINISHED_EVENT, function(context) {
```

Comment:
Please check the correctness of this changed method.
_Previous code is commented with block-commenting_